### ISSUE TITLE
Setup on Docker repo

### ISSUE THREAD

**Issue by mitra42** [mitra42/bct-server#97]
Dan - I've not setup a docker file on any kind of public repo - 
Would appreciate a tutorial, or you doing it. 

I think it should be called safe2 or safe2server

**Comment by danaronson** [comment]
`docker run --rm -p 5000:5000 danaronson/safe2server:1.1`

**Comment by mitra42** [comment]
How do you think we should populate that repo - I'm not sure we want to populate it automatically from Git, otherwise we have to take a lot more care with who could merge a PR here.

**Comment by danaronson** [comment]
one way of doing it is to only populate it from either the master branch or
when we tag anything OR just manually.

--dan

**Comment by mitra42** [comment]
We can leave this open as a placeholder till we figure out how we want to do this later - nothing urgent.

**Comment by mitra42** [comment]
@danaronson  can you summarize what you and @jmday worked out with Tim
